scripts/generate-alerting-rtk-apis.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * To generate alerting k8s APIs, run:
+ * `npx rtk-query-codegen-openapi ./scripts/generate-alerting-rtk-apis.ts`
+ */
+
+import { ConfigFile } from '@rtk-query/codegen-openapi';
+import fs from 'fs';
+
+const schemaFile = '../data/alerting/openapi.json';
+
+try {
+  // Check we have the OpenAPI before generating alerting RTK APIs,
+  // as this is currently a manual process
```

Comment:
As mentioned in the description, we currently have to manually save the OpenAPI json to the data folder and generate from there 🤷